parts/linux/cloud-init/nodecustomdata.yml

```diff
@@ -414,4 +414,15 @@ write_files:
   encoding: gzip
   owner: root
   content: !!binary |
-    {{GetVariableProperty "cloudInitData" "customSearchDomainsScript"}}
\ No newline at end of file
+    {{GetVariableProperty "cloudInitData" "customSearchDomainsScript"}}
+
+{{ if EnableBootstrapProfileContainerRegistry -}}
+- path: /etc/containerd/certs.d/mcr.microsoft.com/hosts.toml
+  permissions: "0644"
+  owner: root
+  content: |
+    server = "https://mcr.microsoft.com"
```

Comment:
will it be possible to add one comment for TODO sovereign cloud? since lx and mc are using differernt mcr url